README.md

```diff
@@ -59,10 +62,3 @@ for each module from git and rebuilds the search indexes.
 
 	05 * * * * sites make -f /sites/ss2doc-v2/www/Makefile -C /sites/ss2doc-v2/www update
```

Comment:
@ss23 is this still the correct cron on our docs server? If not can you drop in the correct one for future reference :)